platform/security/sts/security-sts-upbstvalidator/src/main/java/org/codice/ddf/security/validator/uname/UPBSTValidator.java

```diff
@@ -199,26 +206,30 @@ public TokenValidatorResponse validateToken(TokenValidatorParameters tokenParame
         UsernameTokenType usernameTokenType = getUsernameTokenType(usernameToken);
         // Marshall the received JAXB object into a DOM Element
         Element usernameTokenElement = null;
-        try {
-            Set<Class<?>> classes = new HashSet<Class<?>>();
-            classes.add(ObjectFactory.class);
-            classes.add(
-                    org.apache.cxf.ws.security.sts.provider.model.wstrust14.ObjectFactory.class);
-
-            JAXBContextCache.CachedContextAndSchemas cache = JAXBContextCache
-                    .getCachedContextAndSchemas(classes, null, null, null, false);
-            JAXBContext jaxbContext = cache.getContext();
-
-            Marshaller marshaller = jaxbContext.createMarshaller();
-            Document doc = DOMUtils.createDocument();
-            Element rootElement = doc.createElement("root-element");
-            JAXBElement<UsernameTokenType> tokenType = new JAXBElement<UsernameTokenType>(
-                    QNameConstants.USERNAME_TOKEN, UsernameTokenType.class, usernameTokenType);
-            marshaller.marshal(tokenType, rootElement);
+
+        JAXBElement<UsernameTokenType> tokenType = new JAXBElement<>(QNameConstants.USERNAME_TOKEN,
+                UsernameTokenType.class,
+                usernameTokenType);
+
+        Document doc = DOMUtils.createDocument();
+        Element rootElement = doc.createElement("root-element");
+
+        List<String> ctxPath = new ArrayList<>(1);
+        ctxPath.add(UsernameTokenType.class.getPackage()
+                .getName());
+
+        if (null == parser) {
+            throw new IllegalStateException("XMLParser must be configured.");
+        } else {
+            ParserConfigurator configurator = parser.configureParser(ctxPath,
+                    UPBSTValidator.class.getClassLoader());
+            try {
+                parser.marshal(configurator, tokenType, rootElement);
+            } catch (ParserException ex) {
+                LOGGER.warn("", ex);
+                return response;
+            }
             usernameTokenElement = (Element) rootElement.getFirstChild();
```

Comment:
Move this line into the try block so it reads better and is consistent with `UsernameTokenValidator`.